initfiles/componentfiles/configxml/esp.xsd.in

```diff
@@ -1012,6 +1012,13 @@
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
+            <xs:attribute name="txSummaryStyle" type="xs:nonNegativeInteger" use="optional" default="text">
```

Comment:
"text" is not a non-negative integer. What is the difference between using "default" and "hpcc:presetValue"?